pytorch_lightning/callbacks/quantization.py

```diff
@@ -252,7 +252,7 @@ def _prepare_model(self, model: torch.nn.Module) -> None:
             model.qconfig = self._qconfig
 
         if self._check_feasible_fuse(model):
-            torch.quantization.fuse_modules(model, self._modules_to_fuse, inplace=True)
+            torch.ao.quantization.fuse_modules_qat(model, self._modules_to_fuse, inplace=True)
```

Comment:
The linked issue says: 

> Once torch maps torch.ao.quantization.fuse_modules_qat to torch.quantization.fuse_modules_qat, code should be refactored

Do you know when this will happen?

Reply:
> The linked issue says:
> 
> > Once torch maps torch.ao.quantization.fuse_modules_qat to torch.quantization.fuse_modules_qat, code should be refactored
> 
> Do you know when this will happen?

No clue, sorry, even the part that it will be remapped by torch is a guess on my part, refactored the issue